.github/workflows/showcaseTests.yaml

```diff
@@ -0,0 +1,48 @@
+name: Showcase Tests for Spring Auto-Configurations
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - 'spring-cloud-generator/**'
+  workflow_dispatch:
+
+jobs:
+  showcaseTests:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Java 17
+        uses: actions/setup-java@v1
+        with:
+          java-version: 17
+      - name: Install buildozer
+        run: |
+          go install github.com/bazelbuild/buildtools/buildozer@latest
+          export PATH=$PATH:$(go env GOPATH)/bin
+          buildozer --version
+      - name: Mvn install
```

Comment:
Can you clarify what is being built and installed? Is it just the generator or also the showcase?

Reply:
Here it is the spring-cloud-gcp project from root (local snapshot version, for [the generator](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/6364f896d728eb32a2f8fe8c7ce45f506f81f80d/pom.xml#L454) and dependencies such as spring-cloud-gcp-core). It does not include the showcase client (or showcase spring starters) which are installed through the [script](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1957/files#diff-8b45adb839b5d665b9765b501c4d48cfa965ad0612dcbd0c161ceebd5f38837dR64-R66). I'll add a comment to clarify.